mlflow/pyfunc/__init__.py

```diff
@@ -2179,6 +2180,10 @@ def _save_model_with_loader_module_and_data_path(
         python_env=_PYTHON_ENV_FILE_NAME,
         model_config=model_config,
     )
+    try:
+        mlflow_model.model_size_bytes = get_total_file_size(str(path))
```

Comment:
Can we update the `Model` class to add `model_size_bytes`?

Reply:
Adding an attribute after object construction like this is error-prone and can lead to AttributeError easily.

Reply:
pinged you in the offline discussion thread